java/src/jmri/jmrit/display/layoutEditor/LayoutTrack.java

```diff
@@ -201,7 +196,6 @@ protected boolean isConnectionType(int connectionType) {
      */
     public Point2D getCoordsForConnectionType(int connectionType) {
         log.error("virtual method: override in sub-classes and don't call super...].");
```

Comment:
If this method *must* be overridden by a subclass, and does nothing useful at this point, make it abstract. That will force a compiler error if the method is not implemented in subclasses and eliminate the need for this particular method body.